src/Integration.UnitTests/LocalServices/ProjectSystemFilterTests.cs

```diff
@@ -346,6 +369,15 @@ private ProjectSystemFilter CreateTestSubject()
             return new ProjectSystemFilter(host);
         }
 
+        private void CheckProjectIsNotAccepted(ProjectMock project)
+        {
+            var testSubject = this.CreateTestSubject();
+
+            var result = testSubject.IsAccepted(project);
+
+            result.Should().BeFalse();
```

Comment:
I think this method is not generally needed, it saves two lines and IMO the code in the tests is harder to understand